libs/sdk-core/src/lib/db/resource.ts

```diff
@@ -29,6 +29,7 @@ import { search } from './search';
 export interface ReadableResource extends IResource {}
 export class ReadableResource implements IResource {
   data: ResourceData = {};
+  resourceLabels: SimpleLabel[] = [];
```

Comment:
It looks more like something which helps in a specific use case (=our dashboard) than something people would need when using the SDK.

Reply:
Right, I reworked my commit to avoid this